pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -716,7 +714,12 @@ func sendCtxInit(ctx *logmanagerContext) {
 	zedcloudCtx.TlsConfig = tlsConfig
 	zedcloudCtx.FailureFunc = zedcloud.ZedCloudFailure
 	zedcloudCtx.SuccessFunc = zedcloud.ZedCloudSuccess
-	zedcloudCtx.NetworkSendTimeout = ctx.globalConfig.NetworkSendTimeout
+
+	item, ok := ctx.configItemValueMap.GlobalSettings["timer.port.timeout"]
```

Comment:
You are not supposed to look into GlobalSettings directly. Instead, you should look use the appropriate accessor method ( ItemIntValue() )

zedcloudCtx.NetworkSendTimeout = ctx.configItemValueMap.ItemIntValue("timer.port.timeout")